### DISCUSSION TITLE
Understanding the WorkZoneRoad Event and Associated Lanes

### DISCUSSION THREAD

**Discussion by devorah-ql** [usdot-jpo-ode/jpo-wzdx#348] [Q&A]
I am trying to understand the WorkZone Road Event and associated Lane data, and could use some clarification.  I am basically just trying to understand where the appropriate place is to define the bounds of a work zone, and how to associate it with lane data within that work zone.   

It appears to me that the WorkZone Road Event would be the appropriate level to define a specific work zone, to identify for example that there is roadwork along a specific roadway from one point to another point.  However, when I look at the Lane definitions for a WorkZoneRoadEvent, it appears to me that if the Lane configuration changes within that work zone, then this would require a new WorkZoneRoadEvent.  If that is the case, then the WorkZoneRoadEvent does not define a work zone at all, it just defines one of many segments within a workzone (as there may be many places where the lane configuration of the same road changes, perhaps because a highway goes from 5 to 4 lanes, and even moreso, if exit and entry lanes are included in the lane definition, as these may change often).

So, basically my question is, is the WorkZoneRoadEvent the proper place to define a specific work zone?  If so, I would suggest that multiple lane configurations can be included as an array within the same WorkZoneRoadEvent, so the road configuration may change, but it is the same workzone.  

Or if the WorkZoneRoadEvent is not where a work zone is defined, where does that happen?  When I look at the Object Diagram (copied below), I do not see a parent object that would allow this.  

I know it is also possible I am not understanding the structure of the feed in general, so please feel free to clarify the intended use.  I am interested in understanding the general intent of how work zones and lanes are supposed to be defined here.  Thanks for the assistance.

<img width="1070" alt="image" src="https://user-images.githubusercontent.com/89944134/194440259-d109d513-13ca-485c-9afb-5a89dba32f38.png">

**Comment by Dunge** [comment]
I suggest you refer at the "business rules" and "relationship" section of this document: https://github.com/usdot-jpo-ode/wzdx/blob/main/Creating_a_WZDx_Feed.md

In short, yes a single workzone can be composed of multiple road events when lane geometry changes, and they are linked together via relationships.
